### PR TITLE
misc fixes after upgrade to d3fc v12

### LINKED ISSUE #47: bug on example html page
when displayed on browser, relativeStrengthIndex.html will shows with a horizontal line like "-----[object object]", and lots of error message `d3.js:7672 Error: <g> attribute transform: Trailing garbage, "translate(0, NaN)".` is shown on console.

### PR DESCRIPTION
**Pull request by chrisprice**
Fixes #47

### CODE REVIEW

File: examples/bollingerBands.html [hunk 1]
```diff
@@ -2,16 +2,12 @@
     <head>
         <script src="../node_modules/babel-polyfill/dist/polyfill.js"></script>
         <script src="../node_modules/d3/build/d3.js"></script>
-        <script src="../node_modules/d3fc-rebind/build/d3fc-rebind.js"></script>
-        <script src="../node_modules/d3fc-data-join/build/d3fc-data-join.js"></script>
-        <script src="../node_modules/d3fc-extent/build/d3fc-extent.js"></script>
-        <script src="../node_modules/d3fc-random-data/build/d3fc-random-data.js"></script>
-        <script src="../node_modules/d3fc-series/build/d3fc-series.js"></script>
+        <script src="../node_modules/d3fc/build/d3fc.js"></script>
         <script src="../build/d3fc-technical-indicator.js"></script>
```
Comment (ColinEberhardt) [inline examples/bollingerBands.html]: I'm assuming that including this script after d3fc will cause the local d3fc-technical-indicator build to replace the objects 'registered' to the `fc` namespace by the d3fc bundle?

Comment (chrisprice) [inline examples/bollingerBands.html]: That was also my (untested) assumption but from looking at the generated source it seems to hold. All functions are assigned to the following when a module system isn't detected -
```js
global.fc = global.fc || {}
```

**Comment by chrisprice** [comment]
Good point about the screenshots, I put them in as much to prove that they hadn't changed - will remove 👍

File: examples/bollingerBands.html (hunk 1 above)
Comment (ColinEberhardt) [inline examples/bollingerBands.html]: Cool - I also assumed that it would work, but worth checking!

**Pull request merged**